tests/test_analytics/test_all_analytics_query.py

```diff
@@ -0,0 +1,31 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.analytics.query_all_analytics_fixtures import (
+    all_analytics_query,
+    all_analytics_query_response,
+    analytics_query_for_date_ranges
+)
+
+
+class TestAllAnalytics(BaseTestCase):
+
+    def test_all_analytics_query(self):
+        """
+        Tests a user can query for analytics
+        """
+
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            all_analytics_query,
+            all_analytics_query_response
+        )
+
+    def test_analytics_query_for_date_range(self):
+        """
+        Tests the date range
```

Comment:
Add a more descriptive docstring for your tests

Reply:
Done